dali/ft/filecopy.cpp

```diff
@@ -88,6 +88,8 @@
 #define FAcrc               "@fileCrc"
 #define FAsize              "@size"
 #define FAcompressedSize    "@compressedSize"
+#define FAheaderLength      "@headerLength"
+#define FAfooterLength      "@footerLength"
```

Comment:
Minor: These definitions are duplicates (with subtly difference names (FA* vs FP*) of the definitions in ftbase.cpp. They should be commoned up somewhere and used in-place of the remaining places that use e.g. '@headerLength' directly.